Cubical/Algebra/BooleanRing/Initial.agda

```diff
@@ -0,0 +1,69 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Algebra.BooleanRing.Initial where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Structure
+open import Cubical.Foundations.Function
+
+open import Cubical.Algebra.BooleanRing.Base
+open import Cubical.Data.Bool renaming (elim to bool-ind)
+open import Cubical.Algebra.CommRing
+open import Cubical.Tactics.CommRingSolver
+
+module _ where
+  open BooleanStr
+
+  BoolBRStr : BooleanStr Bool
+  BoolBRStr .𝟘 = false
+  BoolBRStr .𝟙 = true
+  BoolBRStr ._+_ = _⊕_
+  BoolBRStr ._·_ = _and_
+  BoolBRStr .-_ x = x
+  BoolBRStr .isBooleanRing = record
+    { isCommRing = makeIsCommRing isSetBool ⊕-assoc ⊕-identityʳ
+              (bool-ind refl refl) ⊕-comm and-assoc and-identityʳ
+              (bool-ind (λ _ _ → refl) λ _ _ → refl) and-comm
+    ; ·Idem = bool-ind refl refl
+    }
+
+BoolBR : BooleanRing ℓ-zero
+BoolBR = Bool , BoolBRStr
+
+BoolCR : CommRing ℓ-zero
+BoolCR = BooleanRing→CommRing BoolBR
```

Comment:
Shouldn't this go in https://github.com/agda/cubical/tree/master/Cubical/Algebra/CommRing/Instances ?